shared/src/test/scala/mouse/BooleanSyntaxTest.scala

```diff
@@ -9,6 +9,11 @@ class BooleanSyntaxTest extends MouseSuite {
     assertEquals(false.option(1), Option.empty[Int])
   }
 
+  test("BooleanSyntax.option") {
```

Comment:
It would be good to have tests for the `pureOrEmpty` itself.

Reply:
This does test the `pureOrEmpty` method itself. But I am going to rename the test.

Reply:
Done